docs/core_tenets.md

```diff
@@ -0,0 +1,18 @@
+## Jujutsu's Core Tenets
+
+Jujutsu's core tenets are:
+
+ * User-friendliness: Making the  working copy a commit is simpler. This is
+ how the project started.
+ * The "repository", so the commit graph is the source of truth. The working
+ copy is just one way of editing commits.
+ * All operations must be able to scale to Google-scale repos (lots of commits
+ , lots of files): Laziness is important, must avoid accessing data
+ unnecessarily.
+ * Having as few states as possible.
+ * Make it incredibily hard to lose work in your repository.
+ * Allow concurrent edits on any commit, pending or finished.
```

Comment:
What do you mean by this? I think we should rephrase it somehow but I'm not sure what it means. Is it about the operation log? I'd phrase it something like "Concurrent modifications to the repo should be safe.".

Reply:
> What do you mean by this?

This is based on my [statement on Discord][dc]. Since its incredibly easy to revive a leaf commit and update to it land in `trunk()`. 

> Is it about the operation log? I'd phrase it something like "Concurrent modifications to the repo should be safe.".

Not really, but I don't mind adding the latter statement. 

[dc]: https://discord.com/channels/968932220549103686/968932220549103689/1207800007768539167